src/main/webapp/WEB-INF/views/addDriverToCar.jsp

```diff
@@ -0,0 +1,21 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: 38067
+  Date: 25.11.2022
+  Time: 14:54
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<h1>Form to adding a driver to a car</h1>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    Car id: <input type="text" name="car_id"><br>
+    Driver id: <input type="text" name="driver_id"><br>
+    <button type="submit">Confirm</button>
+</form>
```

Comment:
<img width="789" alt="image" src="https://user-images.githubusercontent.com/58032907/204334866-095892bc-0ccf-4367-9f3e-fb4ea6ae4cb1.png">
type should be `number`